src/models/spaces/populations/mouse-model/mice.ts

```diff
@@ -68,7 +68,7 @@ export function getMouseSpecies(model: MousePopulationsModelType) {
       this.set("age", Math.round(Math.random() * 10));
 
       if (this.organism) {
-        if (!model["inheritance.breedWithoutInheritance"]) {
+        if (!model["inheritance.breedWithInheritance"]) {
```

Comment:
This boolean needs to be flipped now that the meaning has changed. (i.e. the `!` needs to be removed.)

Reply:
Pushed a change to fix this.

Reply:
I also rebased into master so we will be ready to merge.